### DISCUSSION TITLE
static assets in v2?

### DISCUSSION THREAD

**Discussion by cekvenich2** [parcel-bundler/parcel#6860] [Q&A]
I get http://localhost:9090/favicon.ico 404 (Not Found)
when I build for prod.

How do I list folders and files that should be copied?

**Comment by cekvenich2** [comment]
never mind found the answer

**Reply by RQDineshkumar** [reply]
@cekvenich2 And what is the answer...?

**Reply by GmBodhi** [reply]
Probably https://github.com/elwin013/parcel-reporter-static-files-copy

**Reply by northern-mindset** [reply]
It is quite inflexible though. Especially in comparison to the v1-compatible https://github.com/elwin013/parcel-plugin-static-files-copy

**Reply by GmBodhi** [reply]
> It is quite inflexible though. Especially in comparison to the v1-compatible https://github.com/elwin013/parcel-plugin-static-files-copy

Is there any alt then?

**Reply by northern-mindset** [reply]
I'm not aware of. Perhaps to improve the elwin013/parcel-reporter-static-files-copy by contributing to it :)

Actually I'm currently resorting to copying the favicon after parcel's build, in the npm script.
